resources/js/Pages/Categories/Index.vue

```diff
@@ -48,8 +48,9 @@
                         </Link>
                     </li>
                 </ul>
-
-                <Link :href="route('categories.create')" v-else
+            </div>
```

Comment:
If there are no products create we show the big white rectangle with the icon as below:
![Skärmavbild 2022-07-06 kl  16 23 40](https://user-images.githubusercontent.com/11759765/177573159-9f6d9d2a-91e7-4544-915c-80c8af0e4a55.png)

However, if there are products we should show the list of products and a button top right with "Create Product" and not show the big white rectangle anymore. The big rectangle is an "empty state".